docs/source/packs.rst

```diff
@@ -235,6 +235,63 @@ than personal access tokens and can be configured on the per-repo basis.
 Other git hosting services should also support either SSH or HTTPS auth,
 and would be configured in a similar fashion.
 
+Installing packs from behind a proxy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you're running |st2| server behind a HTTP/HTTPS proxy, you have to configure |st2| so you
+can install packs from HTTP/HTTPS URLs from behind the proxy. You can setup the proxy
+configuration either by setting the configuration for "packs" pack in |st2| or using environment
+variables for ``st2actionrunner`` process.
+
+Proxy configuration via ``packs`` configuation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Simply drop the proxy configuration in ``/opt/stackstorm/configs/packs.yaml``.
+If you are using HTTP proxy, the configuration looks as follows:
+
+.. code-block:: yaml
+
+    ---
+    http_proxy: "http://proxy.server.io:port"
+    no_proxy: localhost,127.0.0.1
+
+If you are using HTTPS proxy with a proxy CA bundle, then use the following configuration.
+
+.. code-block:: yaml
+
+    ---
+    https_proxy: "https://proxy.server.io:port"
+    proxy_ca_bundle_path: "/etc/ssl/certs/proxy-ca.pem"
+    no_proxy: localhost,127.0.0.1
+
+Proxy configuration via environment vairables for ``st2actionrunner`` process
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can edit ``/etc/default/st2actionrunner`` or the appropriate init file for your OS to
```

Comment:
By convention, `/etc/default/st2actionrunner` for Debian family system and `/etc/sysconfig/st2actionrunner` for RedHat family system.

- `deb` https://github.com/StackStorm/st2-packages/blob/master/packages/st2/debian/st2actionrunner%40.service#L13
- `rpm` https://github.com/StackStorm/st2-packages/blob/master/packages/st2/rpm/st2actionrunner%40.service